core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -462,4 +480,45 @@ public void unmount(AlluxioURI path, UnmountOptions options)
       mFileSystemContext.releaseMasterClient(masterClient);
     }
   }
+
+  /**
+   * Alluxio URIs in the Filesystem don't need a scheme or host. Warn the user and throw an
+   * exception if necessary.
+   */
+  protected static void UriCheck(AlluxioURI uri) {
+    boolean logWarning = false;
+    if (uri.hasScheme()) {
+      if (uri.getScheme().compareTo("alluxio") == 0
```

Comment:
is this different from `.equals()`?

Reply:
functionally, no. I did change it to `.equals()` for better readability though.